src/config/feature-flags.ts

```diff
@@ -89,4 +90,9 @@ export const shouldTagBackfillRequests = async (): Promise<boolean> => {
 	return booleanFlag(BooleanFlags.TAG_BACKFILL_REQUESTS, false);
 };
 
+export const shouldEnableServiceAssociationForDeployments = async (): Promise<boolean> => {
+	logger.error(booleanFlag(BooleanFlags.SERVICE_ASSOCIATIONS_FOR_DEPLOYMENTS, false));
```

Comment:
Curious why do we have this log line?

Reply:
It was for debugging purposes but already removed it